tests/drivers/watchdog/wdt_error_cases/src/main.c

```diff
@@ -42,8 +42,12 @@
 #define DEFAULT_WINDOW_MIN (0U)
 
 /* Align tests to the specific target: */
-#if defined(CONFIG_SOC_NRF54L05) || defined(CONFIG_SOC_NRF54L10) || \
```

Comment:
I think nRF54L symbols can be changed to CONFIG_SOC_NRF54LX.
See:https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/nordic/Kconfig.soc#L48

Reply:
Used `CONFIG_SOC_SERIES_NRF54LX` instead.